connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerConfig.java

```diff
@@ -58,15 +58,12 @@ public class WorkerConfig extends AbstractConfig {
     private static final Logger log = LoggerFactory.getLogger(WorkerConfig.class);
 
     public static final String BOOTSTRAP_SERVERS_CONFIG = "bootstrap.servers";
-    public static final String BOOTSTRAP_SERVERS_DOC
```

Comment:
Since it's the same doc than `CommonClientConfigs`, I think we could reuse `CommonClientConfigs.BOOTSTRAP_SERVERS_DOC` (and do the same for `BOOTSTRAP_SERVERS_CONFIG` above).

Reply:
@mimaison Ah, sorry. I misinterpreted the comment. I see it's merged, so assuming okay for the description change at least.

Reply:
Yeah I decided to keep this change focused on the description change. We can do the refactoring in a follow up PR.